[2024-07-07T20:33:52.316+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-07T20:33:52.333+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl backfill__2019-03-01T00:00:00+00:00 [queued]>
[2024-07-07T20:33:52.338+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl backfill__2019-03-01T00:00:00+00:00 [queued]>
[2024-07-07T20:33:52.339+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-07-07T20:33:52.348+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2019-03-01 00:00:00+00:00
[2024-07-07T20:33:52.353+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=179) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-07T20:33:52.354+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'backfill__2019-03-01T00:00:00+00:00', '--job-id', '513', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmphqua_3is']
[2024-07-07T20:33:52.354+0000] {standard_task_runner.py:63} INFO - Started process 181 to run task
[2024-07-07T20:33:52.354+0000] {standard_task_runner.py:91} INFO - Job 513: Subtask blunders_etl
[2024-07-07T20:33:52.389+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl backfill__2019-03-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-07T20:33:52.444+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='backfill__2019-03-01T00:00:00+00:00'
[2024-07-07T20:33:52.445+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-07T20:33:52.445+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-07T20:33:52.450+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-07T20:34:01.932+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-07T20:34:01.933+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-07T20:34:01.939+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=backfill__2019-03-01T00:00:00+00:00, execution_date=20190301T000000, start_date=20240707T203352, end_date=20240707T203401
[2024-07-07T20:34:01.987+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-07T20:34:01.997+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-07T20:34:01.999+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
